Champignon salad.md

```diff
@@ -6,27 +6,29 @@
 
 ---
 
-Base:
+## Base:
```

Comment:
[Ingredient group titles have to be headings.](https://recipemd.org/specification.html#ingredient-group)